### PR TITLE
Fix Checkstyle plugin test to be CC compatible

### PR DESCRIPTION
**Pull request by tresat**

### CODE REVIEW

**Comment by tresat** [comment]
@bot-gradle test this

**Review by big-guy — APPROVED** [review]

File: subprojects/code-quality/src/integTest/groovy/org/gradle/api/plugins/quality/checkstyle/CheckstylePluginIntegrationTest.groovy [hunk 1]
```diff
@@ -33,20 +33,20 @@ class CheckstylePluginIntegrationTest extends WellBehavedPluginTest {
      */
```
Comment (abstratt) [inline subprojects/code-quality/src/integTest/groovy/org/gradle/api/plugins/quality/checkstyle/CheckstylePluginIntegrationTest.groovy:33]: You would want to remove the blurb above, right?

File: subprojects/code-quality/src/integTest/groovy/org/gradle/api/plugins/quality/checkstyle/CheckstylePluginIntegrationTest.groovy [hunk 2]
```diff
@@ -33,20 +33,20 @@ class CheckstylePluginIntegrationTest extends WellBehavedPluginTest {
...
     def setup() {
         buildFile << """
-            apply plugin: 'groovy'
+            apply plugin: 'java'
+            apply plugin: 'checkstyle'
```
Comment (abstratt) [inline subprojects/code-quality/src/integTest/groovy/org/gradle/api/plugins/quality/checkstyle/CheckstylePluginIntegrationTest.groovy:37]: Note that the inherited test cases all apply the plugin under test automatically (some, qualified/unqualified, some only to an included build). 

Isn't this adding a second application? Is that intentional? I would leave the explicit application to the one test case in this class.

Also, I don't get the switch from groovy to java.

❓

File: subprojects/code-quality/src/integTest/groovy/org/gradle/api/plugins/quality/checkstyle/CheckstylePluginIntegrationTest.groovy (hunk 1 above)
Comment (tresat) [reply subprojects/code-quality/src/integTest/groovy/org/gradle/api/plugins/quality/checkstyle/CheckstylePluginIntegrationTest.groovy:33]: I missed this.  Yes, I will remove.

File: subprojects/code-quality/src/integTest/groovy/org/gradle/api/plugins/quality/checkstyle/CheckstylePluginIntegrationTest.groovy (hunk 2 above)
Comment (tresat) [reply subprojects/code-quality/src/integTest/groovy/org/gradle/api/plugins/quality/checkstyle/CheckstylePluginIntegrationTest.groovy:37]: I'll remove the explicit `checkstyle` application then.

My thought was that groovy plugin applies the java plugin, which is all that is needed to demonstrate the behavior, so why use groovy.

Comment (abstratt) [reply subprojects/code-quality/src/integTest/groovy/org/gradle/api/plugins/quality/checkstyle/CheckstylePluginIntegrationTest.groovy:37]: I will defer to you on that one, Tom, seems reasonable to me

**Review by abstratt — APPROVED** [review]

**Comment by tresat** [comment]
@bot-gradle test and merge

**Pull request merged**